internal/services/pam/pam_test.go

```diff
@@ -486,21 +486,19 @@ func TestSetDefaultBrokerForUser(t *testing.T) {
 
 	tests := map[string]struct {
 		// These are the function arguments.
-		username string
-
-		// These are auxiliary inputs that affect the test setup and help control the mock output.
-		noBroker    bool
-		localBroker bool
+		username     string
+		wantBrokerID string
```

Comment:
it’s not really a `want` (I didn’t like the wantID already), as it’s the argument to the request and we are not really asserting on it (only in golden files), so it’s just brokerID, no?

Reply:
We are asserting on it, but I agree that the naming is off.